src/main/java/org/opentripplanner/framework/retry/OtpRetry.java

```diff
@@ -59,12 +59,7 @@ public void execute(Runnable retryable) {
           sleepTime
         );
         onRetry.run();
-        try {
-          Thread.sleep(sleepTime);
-        } catch (InterruptedException ex) {
-          Thread.currentThread().interrupt();
-          throw new OtpRetryException(ex);
-        }
+        Thread.sleep(sleepTime);
```

Comment:
Isn't this going to block the update thread potentially for a long time?

Reply:
I know that it was like that beforehand so I approve.

Reply:
Yes, it does, but the updater cannot do anything useful without the PubSub subscription anyway, and we don't want the OTP instance to start without the realtime data.
Moreover this blocks only this updater thread, not the common GraphWriter thread.
It is also a non-polling updater, so this does not affect the PollingUpdater thread pool.

Reply:
Thanks for the explanation.